### PR TITLE
[TRANSLATION] Hindi translation of shellext.

### PR DESCRIPTION
**Pull request by arnavbhatt288**
## Purpose

Hindi translation of shellext.

## Proposed changes

- Created hi-IN.rc for every component of shellext (which could get translated)
- Defined it on the resource file of each component.

*phew there is lots of things to do*

### CODE REVIEW

**Comment by arnavbhatt288** [comment]
Found the problem :P

File: dll/shellext/devcpux/lang/hi-IN.rc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    LTEXT "मॉडल:", -1, 7, 49, 50, 8
+    LTEXT "<परीक्षा>", IDC_MODEL, 7, 60, 238, 8
+
+    LTEXT "स्टिपिंग:", -1, 7, 75, 50, 8
+    LTEXT "<परीक्षा>", IDC_STEPPING, 7, 86, 238, 8
+
+    LTEXT "कोर गति:", -1, 7, 101, 50, 8
+    LTEXT "<परीक्षा>", IDC_CORESPEED, 7, 112, 238, 8
+END
+*/
```
Comment (HBelusca) [inline dll/shellext/devcpux/lang/hi-IN.rc]: Please add an extra newline.

File: dll/shellext/devcpux/lang/hi-IN.rc [hunk 2]
```diff
@@ -0,0 +1,41 @@
+LANGUAGE LANG_HINDI, SUBLANG_HINDI_INDIA
```
Comment (HBelusca) [inline dll/shellext/devcpux/lang/hi-IN.rc]: No licensing file header (as done in the other files)?

File: dll/shellext/acppage/lang/hi-IN.rc [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/*
+ * PROJECT:     ACPPAGE Hindi Translation
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```
Comment (learn-more) [inline dll/shellext/acppage/lang/hi-IN.rc]: `GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)` seems to be the preferred style.

**Comment by arnavbhatt288** [comment]
I am still alive, *kinda*

**Pull request merged**